tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_05.bal

```diff
@@ -209,67 +106,8 @@ service mock05 bind backendEP05 {
         path: "/"
     }
     mockResource(endpoint caller, http:Request req) {
-        counter05++;
-        if (counter05 % 5 == 0) {
-            runtime:sleep(30000);
-        }
         http:Response response = new;
-        if (req.hasHeader(mime:CONTENT_TYPE)
```

Comment:
What is the reason for the removal of this test case? Is there an alternative for this test?

Reply:
Yes, We do have separate bal files for different tests to improve the tests execution time. For this particular test, we do not need this logic as those are redundant. 

Multipart tests[1] are not removed from the resiliency tests but the required service is in a separate bal file[2]. Just removed the duplicate codes which not required for this test.

[1]. https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/service/resiliency/HttpResiliencyTest.java#L86

[2]. https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-integration-test/src/test/resources/resiliency/resiliencyservices/http_failover_01.bal#L217